oauth2client/contrib/appengine.py

```diff
@@ -210,6 +211,35 @@ def create_scoped_required(self):
     def create_scoped(self, scopes):
         return AppAssertionCredentials(scopes, **self._kwargs)
 
+    def sign_blob(self, blob):
+        """Cryptographically sign a blob (of bytes).
```

Comment:
I'm not sure why Java wisdom applies here? If someone shows up on [the `appengine` docs][1] page and the doc says "haha, you suck, go [somewhere else][2]", isn't that a negative experience?

PS It seems the GAE docs are back to not building. Seems to be flaky within RTD, will try to track down.

[1]: http://oauth2client.readthedocs.org/en/latest/source/oauth2client.contrib.appengine.html
[2]: http://oauth2client.readthedocs.org/en/latest/source/oauth2client.client.html